src/HTTPure/Response.purs

```diff
@@ -216,16 +218,18 @@ type ResponseM = Aff Response
 type Response =
   { status :: Status
   , headers :: Headers
+  , multiHeaders :: MultiHeaders
   , writeBody :: HTTP.Response -> Aff Unit
   }
 
 -- | Given an HTTP `Response` and a HTTPure `Response`, this method will return
 -- | a monad encapsulating writing the HTTPure `Response` to the HTTP `Response`
 -- | and closing the HTTP `Response`.
 send :: forall m. MonadEffect m => MonadAff m => HTTP.Response -> Response -> m Unit
-send httpresponse { status, headers, writeBody } = do
+send httpresponse { status, headers, multiHeaders, writeBody } = do
   liftEffect $ Status.write httpresponse status
   liftEffect $ Headers.write httpresponse headers
+  liftEffect $ HTTPure.MultiHeaders.write httpresponse multiHeaders
```

Comment:
Hmm, what happens if we have the same headers in `HTTPure.Headers.Headers` and `HTTPure.MultiHeaders.MultiHeaders`?

Reply:
According to the docs on [`request.setHeader`](https://nodejs.org/api/http.html#requestsetheadername-value), the values will be replaced. I'll add this and the reference to the docs on `HTTPure.Headers.write` and `HTTPure.MultiHeaders.write`.

Reply:
> Should we add a test and maybe something to `docs/Examples` to illustrate the behavior as well?

Can add it to the examples yes, although I'm not sure we can test this, as it depends on the node.js HTTP implementation. If we were to add a test we'd have to mock that, and then we'd just be testing that our mock does what we expect it to do.

Reply:
Actually, we can change `HTTPure.Response.send` so that the `headers` are appended to the `multiHeaders`, so headers that exist in both just get joined as if they were `multiHeaders`. I'll make this change and add a test for it.